metadata-service/factories/src/main/resources/application.yml

```diff
@@ -53,6 +53,10 @@ datahub:
     host: ${DATAHUB_GMS_HOST:localhost}
     port: ${DATAHUB_GMS_PORT:8080}
     useSSL: ${DATAHUB_GMS_USE_SSL:${GMS_USE_SSL:false}}
+
+    # URI instead of above host/port/ssl
+    uri: ${DATAHUB_GMS_URI:#{null}}
```

Comment:
If this is an override we should state the priority for the overrides here (and any docs that reference it).